packages/xo-web/src/xo-app/logs/log-alert-header.js

```diff
@@ -19,12 +24,32 @@ export default [
       }),
   })),
   provideState({
+    effects: {
+      restartFailedVms: () => async (
+        { failedVmsIds: vms },
+        { log: { jobId: id, scheduleId: schedule } }
+      ) => {
+        await runBackupNgJob({
+          id,
+          schedule,
+          vms,
+        })
+      },
+    },
     computed: {
       formattedLog: (_, { log }) => JSON.stringify(log, null, 2),
+      failedVmsIds: (_, { log }) =>
+        log === undefined || !isFailureTask(log) || isEmpty(log.tasks)
```

Comment:
Also, what should be expected if the whole job failed?

Reply:
The button `restart all` is not displayed because it have the same behavior of running a job.

Reply:
Don't you think this would be better from a user point of view?